L1Trigger/L1TGlobal/interface/CorrelationThreeBodyByTwoBodyTemplate.h

```diff
@@ -0,0 +1,154 @@
+#ifndef L1Trigger_L1TGlobal_CorrelationThreeBodyByTwoBodyTemplate_h
+#define L1Trigger_L1TGlobal_CorrelationThreeBodyByTwoBodyTemplate_h
+
+/**
+ * \class CorrelationThreeBodyByTwoBodyTemplate,  motivated by class CorrealtionTemplate
+ *
+ *
+ * Description: L1 Global Trigger correlation with 3-object by 2-object template.
+ *
+ * Implementation:
+ *    <TODO: enter implementation details>
+ *
+ * \author: Vladimir Rekovic
+ *
+ * $Date$
+ * $Revision$
+ *
+ */
+
+// system include files
+#include <string>
+#include <iosfwd>
+
+// user include files
+
+//   base class
+#include "L1Trigger/L1TGlobal/interface/GlobalCondition.h"
+
+#include "L1Trigger/L1TGlobal/interface/GlobalDefinitions.h"
+// forward declarations
+
+// class declaration
+class CorrelationThreeBodyByTwoBodyTemplate : public GlobalCondition {
+public:
+  /// constructor(s)
+  ///   default
+  CorrelationThreeBodyByTwoBodyTemplate();
```

Comment:
These member functions do not appear to be defined anywhere, and that broke UBSAN build, see https://github.com/cms-sw/cmssw/issues/36561